src/assets/global.css

```diff
@@ -1,13 +1,44 @@
+@font-face {
+  font-family: Manrope;
+  src: url(./fonts/Manrope-VariableFont_wght.ttf);
+}
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: Manrope;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+/* Css Global Rules - override default Css*/
+* {
+  box-sizing: border-box;
+}
+h1 {
+  margin: 0;
+}
+h2 {
+  margin: 0;
+}
+h3 {
+  margin: 0;
+}
+h4 {
+  margin: 0;
+}
+h5 {
+  margin: 0;
+}
+h6 {
+  margin: 0;
+}
```

Comment:
You can specify several selectors with comma
`h1, h2, h3, h4, h5, h6 {
  margin: 0;
}`

Reply:
Thank you